R/chartutils.R

```diff
@@ -624,6 +625,11 @@ setAxis <- function(title, side, axisLabels, titlefont,
                 {
```

Comment:
In 779ef70, I've moved the blocks containing `use.auto.ticks` to `setDateTickDistance()` to avoid repetition. I've added some more comments in there. Also, I had a think about what that magic number is doing and I think it should be `n+1` instead. Hopefully this doesn't cause changes, and we see only diffs in charts using small multiples.